docs/tables/aws_cloudwatch_metric_statistic_data_point.md

```diff
@@ -0,0 +1,190 @@
+  sample_count
+from
+  aws_cloudwatch_metric_statistic_data_point
+where
+  namespace = 'AWS/EC2'
+  and metric_name = 'CPUUtilization'
+  and average > 80
+  and timestamp between '2023-03-15T00:00:00Z' and '2023-03-15T13:33:00Z'
+  and period = 86400
+  and dimensions = '[
+    {"Name": "InstanceId", "Value": "i-0dd7043e0f6f0f36d"}
+    ]'::jsonb
+order by
+  timestamp;
+```
+
+### Intervals where a volume exceed 1000 average read ops for a given time frame
+
+```sql
+select
+  jsonb_pretty(dimensions) as dimensions,
+  timestamp,
+  minimum,
+  maximum,
+  average,
+  sum,
+  sample_count
+from
+  aws_cloudwatch_metric_statistic_data_point
+where
+  namespace = 'AWS/EBS'
+  and metric_name = 'VolumeReadOps'
+  and average > 1000
+  and timestamp between '2023-03-15T00:00:00Z' and '2023-03-15T13:33:00Z'
+  and period = 86400
+  and dimensions = '[
+    {"Name": "VolumeId", "Value": "vol-00607053b218c6d74"}
+    ]'::jsonb
+order by
+  timestamp;
+```
+
+### CacheHit sum below 10 of an elasticache cluster for a given time frame
+
+```sql
+select
+  jsonb_pretty(dimensions) as dimensions,
+  timestamp,
+  minimum,
+  maximum,
+  average,
+  sum,
+  sample_count
+from
+  aws_cloudwatch_metric_statistic_data_point
+where
+  namespace = 'AWS/ElastiCache'
+  and metric_name = 'CacheHits'
+  and sum < 10
+  and timestamp between '2023-03-15T00:00:00Z' and '2023-03-15T13:33:00Z'
+  and period = 86400
+  and dimensions = '[
+    {"Name": "CacheClusterId", "Value": "cluster-delete-001"}
+    ]'::jsonb
+order by
+  timestamp;
+```
+
+### Lambda function daily maximum duration over 100 milliseconds for a given time frame
+
+```sql
+select
+  jsonb_pretty(dimensions) as dimensions,
+  timestamp,
+  minimum,
+  maximum,
+  average,
+  sum,
+  sample_count
+from
+  aws_cloudwatch_metric_statistic_data_point
+where
+  namespace = 'AWS/Lambda'
+  and metric_name = 'Duration'
+  and maximum > 100
+  and timestamp between '2023-03-15T00:00:00Z' and '2023-03-15T13:33:00Z'
+  and period = 86400
+  and dimensions = '[
+    {"Name": "FunctionName", "Value": "test"}
+    ]'::jsonb
+order by
+  timestamp;
+```
+
+### CPU average utilization of an RDS DB instance over 80% for a given time frame
+
+```sql
+select
+  jsonb_pretty(dimensions) as dimensions,
+  timestamp,
+  round(minimum::numeric,2) as min_cpu,
+  round(maximum::numeric,2) as max_cpu,
+  round(average::numeric,2) as avg_cpu,
+  sample_count
+from
+  aws_cloudwatch_metric_statistic_data_point
+where
+  namespace = 'AWS/RDS'
+  and metric_name = 'CPUUtilization'
+  and average > 80
+  and timestamp between '2023-03-15T00:00:00Z' and '2023-03-15T13:33:00Z'
+  and period = 86400
+  and dimensions = '[
+    {"Name": "DBInstanceIdentifier", "Value": "database-1"}
+    ]'::jsonb
+order by
+  timestamp;
+```
+
+### Bucket size statistics of an s3 bucket for a given time frame
+
+```sql
+select
+  jsonb_pretty(dimensions) as dimensions,
+  timestamp,
+  minimum,
+  maximum,
+  average,
+  sum,
+  sample_count
```

Comment:
Don't most of these statistics return the same data since the metric only has 1 data point per day per bucket? Which statistic should we use?